.github/workflows/publish-arc.yaml

```diff
@@ -58,6 +58,9 @@ jobs:
             ${{ env.DOCKERHUB_USERNAME }}/actions-runner-controller:latest
             ${{ env.DOCKERHUB_USERNAME }}/actions-runner-controller:${{ env.VERSION }}
             ${{ env.DOCKERHUB_USERNAME }}/actions-runner-controller:${{ env.VERSION }}-${{ steps.vars.outputs.sha_short }}
+            ghcr.io/actions-runner-controller/actions-runner-controller:latest
+            ghcr.io/actions-runner-controller/actions-runner-controller:${{ env.VERSION }}
+            ghcr.io/actions-runner-controller/actions-runner-controller:${{ env.VERSION }}-${{ steps.vars.outputs.sha_short }}
```

Comment:
I guess we might also need the permission for package writing at least, as similar as we do for the canary builds (#1524):

https://github.com/actions-runner-controller/actions-runner-controller/blob/2733c36d0ed30a8761ae8b42636975efaee1ae2c/.github/workflows/publish-canary.yaml#L23-L25

Reply:
we don't have any exlicit permissions set at the moment so the token will be write all. I'll do a little test to figure out which we need and add them to his PR.